index.d.ts

```diff
@@ -131,7 +131,7 @@ declare module 'enzyme' {
     nodeToHostNode(node: RSTNode): Node | null;
 
     // Optional methods.
-    displayNameOfNode?(node: RSTNode): string;
+    displayNameOfNode?(node: RSTNode): string | null;
```

Comment:
It appears this method can return null and should it some cases, so I updated the types: https://github.com/enzymejs/enzyme/blob/9bb92eb6dad19152766549bb22039264982e673b/packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js#L888-L889